pkg/apis/abac/fuzzer/fuzzer.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fuzzer
+
+import (
+	"k8s.io/apimachinery/pkg/api/testing/fuzzer"
+	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+func abacFuncs(codecs runtimeserializer.CodecFactory) []interface{} {
+	return []interface{}{}
+}
+
+// Funcs returns the fuzzer functions for the abac api group.
+var Funcs = fuzzer.MergeFuzzerFuncs(
```

Comment:
Is there a need to call merge here - how about just renaming `abacFuncs` to `Funcs`? Same question for all the other new files as well.